charts/index.yaml

```diff
@@ -1,6 +1,28 @@
 apiVersion: v1
 entries:
   karmada:
+  - apiVersion: v2
```

Comment:
steps to sync this:

```
wget https://github.com/karmada-io/karmada/releases/download/v1.7.1/karmada-chart-v1.7.1.tgz -O charts/karmada/charts/karmada-chart-v1.7.1.tgz
 
helm repo index charts/karmada --url https://github.com/karmada-io/karmada/releases/download --merge charts/index.yaml
 
mv charts/karmada/index.yaml charts/index.yaml 
```

+ manual steps:

1. change appVersion from `latest` to `v1.1.0`
2. change the`urls` from `https://github.com/karmada-io/karmada/releases/download/charts/karmada-chart-v1.7.0.tgz` to `https://github.com/karmada-io/karmada/releases/download/v1.7.1/karmada-chart-v1.7.1.tgz`